src/pages/Dashboard/Portfolio.tsx

```diff
@@ -20,10 +22,13 @@ export const Portfolio = () => {
   const [percentChange, setPercentChange] = useState(0)
 
   const assetIds = useSelector(selectPortfolioAssetIds)
-  const totalBalance = useSelector(selectPortfolioTotalFiatBalance)
+  const totalBalanceFiat = useSelector(selectPortfolioTotalFiatBalance)
+  const totalDeligationFiat = useSelector(selectTotalStakingDelegationFiat)
   const loading = useSelector(selectPortfolioLoading)
   const isLoaded = !loading
 
+  const totalBalance = bnOrZero(totalBalanceFiat).plus(totalDeligationFiat).toString()
```

Comment:
again this kind of logic should go in a selector composed of the others, arguably within the selectPortfolioTotalFiatBalance selector

Reply:
Should it be its own selector? 

It seems useful to have these value separate and its just a "+" of 2 different values

Reply:
ok I added a selector